docs/sources/microsoft-365/ms-teams/ms-teams.yaml

```diff
@@ -0,0 +1,232 @@
+---
+endpoints:
+  - pathRegex: "^/(v1.0|beta)/users/?[^/]*"
```

Comment:
No `users` endpoint used for Teams; same for `Groups` or `GroupMembers`

Reply:
Removed.